functions/src/index.ts

```diff
@@ -0,0 +1,49 @@
+import express from "express";
+import cors from "cors";
+import bodyParser from "body-parser";
+
+import LobWebhookHandler from "./handlers/LobWebhook";
+import StripPaymentWebhookHandler from "./handlers/StripePaymentWebhook";
+import StartPaymentHandler from "./handlers/LobWebhook";
+
+import OrderUpdateTrigger from "./triggers/OrderUpdate";
+
+/** wraps an express requst handler in an express app
+ * we do this to add cors and json body parsing
+ * it mounts it at "/" and then we export it at a named location for firebase to mount there
+ * If we made one big express app, firebase wouldn't understand it as multiple api endpoints
+ * 
+ * @param {express.RequestHandler} handler the handler to wrap
+ * @return {string} the handler wrapped as an express app on /
+ */
+function wrapPostExpressHandler(handler: express.RequestHandler) {
+  const app = express();
+
+  // allow json requests from anywhere for ease of use
+  // these are hosted on cloudfunctions.net but our frontend is on defund12.org
+  app.use(cors({ origin: true }));
+
+  // automatically parse json POST bodies
```

Comment:
nit) remove unnecessary comments.

```
  app.use(bodyParser.json());
  app.post("/", handler);

  return app;
```

Reply:
I'd prefer to leave these, some of the others I'm fine to remove. I was trying to make this very contributor friendly.